.gitignore

```diff
@@ -9,3 +9,7 @@ nosetests.xml
 octodns.egg-info/
 output/
 tmp/
+build/
+config/
```

Comment:
`build/` and `config/` entries probably make sense, but please remove the latter 2.